packages/material-ui-styles/src/getStylesCreator/getStylesCreator.js

```diff
@@ -1,6 +1,25 @@
 import { deepmerge } from '@material-ui/utils';
 import noopTheme from './noopTheme';
 
+const capitalize = (string) => {
```

Comment:
I moved this to `@material-ui/utils` and re-exported this under `material-ui/core/utils`, in order to avoid to many changes of the imports with this PR. We can remove that later if we need to.

Reply:
Seems like there are still circular dependencies, I believe it's because I am importing there: `import MuiError from '../macros/MuiError.macro';` 🤔 How can I fix this?